src/meshcat/visualizer.py

```diff
@@ -134,8 +134,9 @@ def jupyter_cell(self):
     def __getitem__(self, path):
         return Visualizer.view_into(self.window, self.path.append(path))
 
-    def set_object(self, geometry, material=None):
-        return self.window.send(SetObject(geometry, material, self.path))
+    def set_object(self, geometry, material=None, texts=None):
```

Comment:
Can we get rid of the `texts` argument here as well?

Reply:
Done.